docs/distribution/launcher-cli.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: developer-doc
+title: Enso Launcher CLI
+category: distribution
+tags: [distribution, launcher, cli, command]
+order: 5
+---
+
+# Enso Launcher CLI
+This document describes available command-line options of the Enso launcher.
+
+> The actionables for this document are:
+>
+> - This document is a draft. The explanations are just to give an idea of the
+>   commands. It will be updated with detailed explanations when the CLI is
+>   developed.
+
+<!-- MarkdownTOC levels="2,3" autolink="true" -->
+
+- [Commands](#commands)
+  - [`new`](#new)
+  - [`install`](#install)
+  - [`uninstall`](#uninstall)
+  - [`list`](#list)
+  - [`default`](#default)
+  - [`config`](#config)
+  - [`run`](#run)
+  - [`repl`](#repl)
+  - [`language-server`](#language-server)
+  - [`upgrade`](#upgrade)
+  - [`version`](#version)
+  - [`help`](#help)
+- [General Options](#general-options)
+  - [`--path`](#--path)
+  - [`--version`](#--version)
+  - [`--use-system-jvm`](#--use-system-jvm)
+
+<!-- /MarkdownTOC -->
+
+## Commands
+
+### `new`
+Create a new, empty project in a specified directory.
+By default uses the `default` enso version, which can be overriden with
+`--version`.
+
+Examples:
+```bash
+> enso new project1 --path /somewhere/on/the/filesystem
+    # creates project called project1 in the specified directory
+    # using the `default` Enso version
+> enso new project2 --version 2.0.1
+    # creates the project in the current directory, using the 2.0.1 version
+```
+
+### `install`
+Installs a specific version of Enso.
+
+Examples:
+```bash
+> enso install 2.0.1
+```
+
+### `uninstall`
+Uninstalls a specific version of Enso.
+
+Examples:
+```bash
+> enso uninstall 2.0.1
+```
+
+### `list`
+Lists all installed versions of Enso and managed GraalVM distributions.
+
+Optional arguments are `enso` to just list Enso installations or `runtime` to
+list the installed runtimes.
+
+Examples:
+```bash
+> enso list
+Enso 2.0.1 -> GraalVM 20.1.0-java11
+Enso 2.2.3 -> GraalVM 20.1.0-java11
+Enso 3.0.0 -> GraalVM 21.1.0-java14
+> enso list enso
+2.0.1
+2.2.3
+3.0.0
+> enso list runtime
+GraalVM 20.1.0-java11 (used by 2 Enso installations)
+GraalVM 21.1.0-java14 (used by 1 Enso installation)
+```
+
+### `default`
+Sets the default Enso version used outside of projects.
+
+If run without arguments, displays currently configured `default` version.
+
+Examples:
+```bash
+> enso default 2.0.1
+default set to version 2.0.1 
+> enso default
+default version is 2.0.1
+```
+
+### `config`
+Can be used to manage project configuration or global user configuration (if
+outside a project or with the `--global` flag).
+
+If only the config path is provided, currently configured value is printed.
+
+Examples:
+```bash
+> enso config --global user.name Example User
```

Comment:
Why `user.name` here? Do we use user names anywhere? Or should be `author.name` again?